From a958db8f429c7315aa094103197e708f4b98beba Mon Sep 17 00:00:00 2001
From: Raka Prasetya Pratama <madeayurenapramudya@gmail.com>
Date: Mon, 25 Sep 2023 15:06:36 +0000
Subject: [PATCH 2/3] props: remove duplicated adb flasg

---
 config/props.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/props.mk b/config/props.mk
index 8016772..1eaf2ac 100644
--- a/config/props.mk
+++ b/config/props.mk
@@ -25,7 +25,7 @@ ifeq ($(TARGET_BUILD_VARIANT),eng)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 else
 # Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
+# PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-- 
2.37.3

