From e91dedde6614c5bbec3e35c62519200627e15dc0 Mon Sep 17 00:00:00 2001
From: Raka Prasetya Pratama <madeayurenapramudya@gmail.com>
Date: Sun, 30 Apr 2023 06:26:39 +0000
Subject: [PATCH] Disable vendor mismatch warning

---
 .../server/wm/ActivityTaskManagerService.java    | 16 ++--------------
 1 file changed, 2 insertions(+), 14 deletions(-)

diff --git a/services/core/java/com/android/server/wm/ActivityTaskManagerService.java b/services/core/java/com/android/server/wm/ActivityTaskManagerService.java
index bd7a40ac820b..b6abeee3b079 100644
--- a/services/core/java/com/android/server/wm/ActivityTaskManagerService.java
+++ b/services/core/java/com/android/server/wm/ActivityTaskManagerService.java
@@ -5880,20 +5880,8 @@ public class ActivityTaskManagerService extends IActivityTaskManager.Stub {
                 }
 
                 if (!Build.isBuildConsistent()) {
-                    Slog.e(TAG, "Build fingerprint is not consistent, warning user");
-                    mUiHandler.post(() -> {
-                        if (mShowDialogs) {
-                            AlertDialog d = new BaseErrorDialog(mUiContext);
-                            d.getWindow().setType(WindowManager.LayoutParams.TYPE_SYSTEM_ERROR);
-                            d.setCancelable(false);
-                            d.setTitle(mUiContext.getText(R.string.android_system_label));
-                            d.setMessage(mUiContext.getText(R.string.system_error_manufacturer));
-                            d.setButton(DialogInterface.BUTTON_POSITIVE,
-                                    mUiContext.getText(R.string.ok),
-                                    mUiHandler.obtainMessage(DISMISS_DIALOG_UI_MSG, d));
-                            d.show();
-                        }
-                    });
+                    Slog.e(TAG, "Build fingerprint is not consistent");
+                    // Do not emit warning about vendor mismatch
                 }
             }
         }
-- 
2.37.3

