From 530a7e7c4bd7599f24041f69ce4098cd2e552fe1 Mon Sep 17 00:00:00 2001
From: Raka Prasetya Pratama <madeayurenapramudya@gmail.com>
Date: Tue, 28 Mar 2023 00:59:59 +0000
Subject: [PATCH 1/2] build: Don't handle apns-conf

---
 target/product/aosp_product.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/target/product/aosp_product.mk b/target/product/aosp_product.mk
index f651f54f5..83482f98e 100644
--- a/target/product/aosp_product.mk
+++ b/target/product/aosp_product.mk
@@ -40,7 +40,7 @@ PRODUCT_PACKAGES += \
 
 # Telephony:
 #   Provide a APN configuration to GSI product
-ifeq ($(OCTAVI_BUILD),)
-PRODUCT_COPY_FILES += \
-    device/sample/etc/apns-full-conf.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/apns-conf.xml
-endif
+#ifeq ($(OCTAVI_BUILD),)
+#PRODUCT_COPY_FILES += \
+#    device/sample/etc/apns-full-conf.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/apns-conf.xml
+#endif
-- 
2.37.3

