From 3a35b666928641f67059cbd0dcdb59e98cece899 Mon Sep 17 00:00:00 2001
From: Hirozuto <madeayurenapramudya@gmail.com>
Date: Mon, 8 May 2023 05:38:58 +0000
Subject: [PATCH 2/2] build: Remove duplicated ro.adb.secure

---
 core/main.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index 2abd9d2d6..d987759c2 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -371,9 +371,9 @@ ifneq (,$(user_variant))
   ADDITIONAL_SYSTEM_PROPERTIES += ro.secure=1
   ADDITIONAL_SYSTEM_PROPERTIES += security.perf_harden=1
 
-  ifeq ($(user_variant),user)
-    ADDITIONAL_SYSTEM_PROPERTIES += ro.adb.secure=1
-  endif
+#  ifeq ($(user_variant),user)
+#    ADDITIONAL_SYSTEM_PROPERTIES += ro.adb.secure=1
+#  endif
 
   ifeq ($(user_variant),userdebug)
     # Pick up some extra useful tools
-- 
2.40.1

