From fae23bdb096854b902a775e358a40eec344f72a9 Mon Sep 17 00:00:00 2001
From: Raka Prasetya Pratama <madeayurenapramudya@gmail.com>
Date: Fri, 3 Mar 2023 14:18:39 +0000
Subject: [PATCH] Disable optimization for the sse4.2/popcount

---
 build/Android.bp | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/build/Android.bp b/build/Android.bp
index ab90dbe619..335836503f 100644
--- a/build/Android.bp
+++ b/build/Android.bp
@@ -135,22 +135,6 @@ art_global_defaults {
         darwin: {
             enabled: false,
         },
-        host: {
-            cflags: [
-                // Bug: 15446488. We don't omit the frame pointer to work around
-                // clang/libunwind bugs that cause SEGVs in run-test-004-ThreadStress.
-                "-fno-omit-frame-pointer",
-                // The build assumes that all our x86/x86_64 hosts (such as buildbots and developer
-                // desktops) support at least sse4.2/popcount. This firstly implies that the ART
-                // runtime binary itself may exploit these features. Secondly, this implies that
-                // the ART runtime passes these feature flags to dex2oat and JIT by calling the
-                // method InstructionSetFeatures::FromCppDefines(). Since invoking dex2oat directly
-                // does not pick up these flags, cross-compiling from a x86/x86_64 host to a
-                // x86/x86_64 target should not be affected.
-                "-msse4.2",
-                "-mpopcnt",
-            ],
-        },
     },
 
     codegen: {
-- 
2.39.2

