From 0bed9c207e4bf9793687a3f6e898253c6708b50f Mon Sep 17 00:00:00 2001
From: Raka Prasetya Pratama <madeayurenapramudya@gmail.com>
Date: Tue, 28 Mar 2023 01:50:08 +0000
Subject: [PATCH 2/2] build_soong: Disable generated_kernel_headers

---
 build/soong/Android.bp | 33 ---------------------------------
 1 file changed, 33 deletions(-)

diff --git a/build/soong/Android.bp b/build/soong/Android.bp
index dfc69fb7..ca074983 100644
--- a/build/soong/Android.bp
+++ b/build/soong/Android.bp
@@ -17,39 +17,6 @@ bootstrap_go_package {
     pluginFor: ["soong_build"],
 }
 
-octavi_generator {
-    name: "generated_kernel_includes",
-
-    // The headers make command
-    cmd: "$(PATH_OVERRIDE_SOONG) $(KERNEL_MAKE_CMD) $(KERNEL_MAKE_FLAGS) -C $(TARGET_KERNEL_SOURCE) O=$(KERNEL_BUILD_OUT_PREFIX)$(genDir) ARCH=$(KERNEL_ARCH) $(KERNEL_CROSS_COMPILE) headers_install",
-
-    // Directories that can be imported by a cc_* module generated_headers property
-    export_include_dirs: [
-        "usr/audio/include/uapi",
-        "usr/include",
-        "usr/include/audio",
-        "usr/include/audio/include/uapi",
-        "usr/techpack/audio/include",
-    ],
-
-    // Sources for dependency tracking
-    dep_root: "$(TARGET_KERNEL_SOURCE)",
-    dep_files: [
-        "Makefile",
-        "include/**/*",
-        "arch/$(KERNEL_ARCH)/include/**/*",
-        "techpack/audio/include/**/*",
-    ],
-}
-
-cc_library_headers {
-    name: "generated_kernel_headers",
-    generated_headers: ["generated_kernel_includes"],
-    export_generated_headers: ["generated_kernel_includes"],
-    vendor_available: true,
-    recovery_available: true,
-}
-
 soong_config_module_type {
     name: "camera_needs_client_info",
     module_type: "cc_defaults",
-- 
2.40.0

