From d2d3578ec6441d30e5e7188459ce792f499c4d8d Mon Sep 17 00:00:00 2001
From: Raka Prasetya Pratama <madeayurenapramudya@gmail.com>
Date: Tue, 14 Mar 2023 09:57:54 +0000
Subject: [PATCH] Adding missing filesystem

---
 main.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/main.cpp b/main.cpp
index b07ee68..50fb777 100644
--- a/main.cpp
+++ b/main.cpp
@@ -72,6 +72,8 @@ int main(int argc, char** argv) {
     ATRACE_BEGIN("main");
 
     LOG(DEBUG) << "Detected support for:"
+               << (android::vold::IsFilesystemSupported("texfat") ? " texfat" : "")
+               << (android::vold::IsFilesystemSupported("exfat") ? " exfat" : "")
                << (android::vold::IsFilesystemSupported("ext4") ? " ext4" : "")
                << (android::vold::IsFilesystemSupported("f2fs") ? " f2fs" : "")
                << (android::vold::IsFilesystemSupported("vfat") ? " vfat" : "");
-- 
2.37.3

