From cd7684541800b096f5bf424c0b367afc78b75f31 Mon Sep 17 00:00:00 2001
From: Raka Prasetya Pratama <madeayurenapramudya@gmail.com>
Date: Fri, 3 Mar 2023 14:43:25 +0000
Subject: [PATCH 1/2] Add missing texfat

---
 main.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/main.cpp b/main.cpp
index 4d61110..332e3a7 100644
--- a/main.cpp
+++ b/main.cpp
@@ -62,6 +62,7 @@ int main(int argc, char** argv) {
     ATRACE_BEGIN("main");
 
     LOG(DEBUG) << "Detected support for:"
+               << (android::vold::IsFilesystemSupported("texfat") ? " texfat" : "")
                << (android::vold::IsFilesystemSupported("exfat") ? " exfat" : "")
                << (android::vold::IsFilesystemSupported("ext4") ? " ext4" : "")
                << (android::vold::IsFilesystemSupported("f2fs") ? " f2fs" : "")
-- 
2.39.2

